optimism2/erc20/tokens.sql

```diff
@@ -192,6 +192,7 @@ COPY erc20.tokens (contract_address, symbol, decimals) FROM stdin;
 \\xf390830DF829cf22c53c8840554B98eafC5dCBc2	anyUSDC	6
 \\xd6909e9e702024eb93312B989ee46794c0fB1C9D	BICO	18
 \\xd52f94DF742a6F4B4C8b033369fE13A41782Bf44	L2DAO	18
+\\x9485aca5bbBE1667AD97c7fE7C4531a624C8b1ED	agEUR	18
```

Comment:
@MSilb7 fyi that this tokens file was edited today, including this token :) #1311